cmd/context/create.go

```diff
@@ -196,6 +194,34 @@ func (c *ContextCommand) UseContext(ctx context.Context, ctxOptions *ContextOpti
 	return nil
 }
 
+func hasAccessToNamespace(ctx context.Context, c *ContextCommand, ctxOptions *ContextOptions) (bool, error) {
```

Comment:
is this executed at the beginning of each cli command? or only for `okteto context`?

Reply:
only for `okteto context` command

Reply:
should we check access to the namespace on `okteto namespace use xxxx` too?

Reply:
I think so. Maybe the user wants to use a non-existent namespace and we should notify them.

Reply:
well, reviewing your question, a user who does not use a cluster managed by Okteto would never encounter the issue if he runs okteto namespace xxxx.